examples/time_series/goes_xrs_nrt_data.py

```diff
@@ -0,0 +1,71 @@
+"""
+==================================
+GOES XRS Near Real Time (NRT) data
+==================================
+
+Currently, ``sunpy`` has no mechanism to directly download several of the the GOES XRS Near Real Time (NRT) data.
```

Comment:
```suggestion
Currently, ``sunpy`` has no mechanism to directly download the GOES XRS Near Real Time (NRT) data.
```

A small change but I am also unsure on the phrase "no mechanism to", it is probably fine to stay though.

Reply:
I agree. I suggested its reworded to something along the lines of:

```suggestion
NOAA provides the GOES-XRS near-real time data of observations of the past 7 days that is updated ever minute in a json file format. Here we shall show an example of how to load this data and create a `sunpy.timeseries.TimeSeries`. Note that the scientific quality data can be  queried and downloaded through Fido.
```